cmd/krel/cmd/release_notes.go

```diff
@@ -788,7 +788,7 @@ func releaseNotesJSON(repoPath, tag string) (jsonString string, err error) {
 
 	// Preclone the repo to be able to read branches and tags
 	logrus.Infof("Cloning %s/%s", git.DefaultGithubOrg, git.DefaultGithubRepo)
-	repo, err := git.CloneOrOpenDefaultGitHubRepoSSH(repoPath)
+	repo, err := git.CloneOrOpenGitHubRepo(repoPath, git.DefaultGithubOrg, git.DefaultGithubRepo, false)
```

Comment:
Thanks for the change @csantanapr 
What this change will do is avoid cloning the repo via ssh and instead do all the copy operations via https. I'm willing to have this change because I guess members of the release team will bump into fewer problems as cloning with https is simpler.

But I'm concerned that we are not really getting to the root of the problem. If there is a problem with one of the git dependencies below it would be useful to know so that we can fix it.

Any idea what may be causing the key errors?

Reply:
The latest version of git-go doesn't have the fix form it's dependency https://github.com/go-git/go-git/issues/411

I found this on argocd issue https://github.com/argoproj/argo-cd/issues/7723

the Go SSH client also doesn’t yet support RSA with SHA-2, so we recommend using an Ed25519 key there
https://github.blog/2021-09-01-improving-git-protocol-security-github/#libgit2-and-other-git-clients

I believe that's why its having issues despite not using DSA.

Reply:
Hi, @justaugustus I'm not familiar with `kpromo pr` is this is a tool I need to run against this PR to get it merge?

Reply:
> Hi, @justaugustus I'm not familiar with `kpromo pr` is this is a tool I need to run against this PR to get it merge?

@csantanapr -- It's a tool we use for image promotion manifest updates, which will trigger the creation of a PR. I figure you can borrow some logic from it to create the release notes PRs here. :)

Reply:
In other words there is agreement to merge this PR 😉